nearcore/src/runtime/mod.rs

```diff
@@ -264,7 +265,7 @@ impl NightshadeRuntime {
         match chain_id {
             "testnet" => {
                 let genesis_runtime_config =
-                    serde_json::from_slice(INITIAL_TESTNET_CONFIG).unwrap();
+                    RuntimeConfig::from_parameters_txt(INITIAL_TESTNET_CONFIG);
```

Comment:
FWIW, feels like an abstraction leakage. runtime shouldn't care if we store config in json or txt. I'd maybe add `fn RuntimeConfig::initial_testnet_config` instead?